arviz/plots/distplot.py

```diff
@@ -145,6 +146,8 @@ def plot_dist(
 
         >>> az.plot_dist(b, rug=True, quantiles=[.25, .5, .75], cumulative=True)
     """
+    values = np.array(values) if str(type(values)) == "<class 'list'>" else values
```

Comment:
Thank you for the contribution1 

For feedback instead of comparing the string of the object could `isinstance` be used instead? See this stackoverflow answer for further details
https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance

As a matter of personal preference as well could this be expanded to be multi lines. Its clever to keep it on one line but harder to read

Reply:
Just to note this as well, please add a test to the testsuite to double check this functionality! If you need help let us know

Reply:
Sir this was my first contribution, so i want to learn how to add test to the testsuite..

Reply:
Youre doing great! The first PR is the hardest so just bear with the comments and we'll get there. 

We use pytest with arviz. The docs on this website will show what those look like 
https://docs.pytest.org/en/stable/

When you get a handle of how pytest works you'll want to add a test that's similar to this one. The idea is to test the the list conversion works as expected. By writing these tests we can be sure ArviZ functionality is working as intended, both for this PR but also in the future
https://github.com/arviz-devs/arviz/blob/master/arviz/tests/base_tests/test_plots_matplotlib.py#L374

Reply:
I think this line should be replaced with

    values = np.asarray(values)

It does basically the same checks in it, but works with tuples too.